runtime/onert/backend/trix/CMakeLists.txt

```
@@ -1,6 +1,9 @@
 set(LIB_ONERT_BACKEND_TRIX onert_backend_trix)
 
-nnfw_find_package(TRIXEngine QUIET 2.5.0)
+find_package(PkgConfig REQUIRED)
+# TRIXEngine version is required to higher than 2.5.0
+pkg_check_modules(TRIXEngine REQUIRED IMPORTED_TARGET npu-engine>2.5.0)
```

```suggestion
pkg_check_modules(TRIXEngine QUIET IMPORTED_TARGET npu-engine>2.5.0)
```

If it is `REQUIRED`, build fail when `npu-engine` dev package is not installed and below `TRIXEngine_FOUND` check becomes meaningless.

---

Yes you're right, I'll fix it right away.

---

Updated!